_webi/test.js

```diff
@@ -66,7 +66,7 @@ Releases.get(path.join(process.cwd(), pkgdir)).then(function (all) {
   var pkgname = path.basename(pkgdir.replace(/\/$/, ''));
```

Comment:
This commit is already merged into master and shouldn't show up in your list. I wonder what went wrong here.

Reply:
i don't know how but after i did  git pull --rebase https://github.com/webinstall/packages master its showing me recent commits made by other people along with my commit

Reply:
That's what rebase does (it brings you up-to-date and the replays your new commits on top of what's in the most current master). Just rebase again and don't squash those commits this time.

Please also see my other comments in issue comments.